api.yaml

```diff
@@ -322,7 +322,65 @@ paths:
               #! end !#
               "bodyPassthrough": true
             }
-
+  /v1/cloud/arnid/{arnid}:
+    get:
+      summary: "Retrieve a cloud asset at a point in time by ARN ID"
+      parameters:
+        - name: "arnid"
```

Comment:
nit: Please confirm arnid pattern doesn't break url or require character escaping

Reply:
The ARN ID contains ASCII safe characters. For example, "i-123423453b56jdk806". So, it does not require character escaping.